### DISCUSSION TITLE
Translations and Xcode

### DISCUSSION THREAD

**Discussion by sanberkzulfikar** [utmapp/UTM#4545] [General]
Is the only way to do translations to UTM using Xcode?

**Comment by osy** [comment]
@MMP0 might be good to ask

**Comment by MMP0** [comment]
If you want to add a language, you need Xcode. This is because the configuration file `project.pbxproj` must be modified properly when adding to the list of localizable languages or adding files.
If you want to make changes to an existing language translation, you don’t need Xcode. You can simply fork the repository, modify the `.strings` files, and create a PR.

**Reply by sanberkzulfikar** [reply]
What if I request a .strings file for the target new language and fork it after you?

**Reply by MMP0** [reply]
What language would you like to add?
If you upload your localization files in a ZIP file, I’ll create a pull request to add the language and files on your behalf. Once the language and files have been added, you can edit and create PRs in GitHub.

**Reply by sanberkzulfikar** [reply]
Glad it would work. Language I want to translate to is Turkish.

**Reply by sanberkzulfikar** [reply]
What's the current state?

**Reply by sanberkzulfikar** [reply]
Hey?
